### ISSUE TITLE
Improve tauri integration

### ISSUE THREAD

**Issue by joepio** [joepio/atomic-data-browser#115]
I just finished running Atomic-Server in tauri: ihttps://github.com/joepio/atomic-data-rust/issues/215

But some front-end stuff could be improved

- [ ] Show back / forward buttons while running in Tauri
- [ ] Explore cool features

**Comment by iilyak** [comment]
I just came across atomic. Very cool project. I was searching github how to integrate tauri and actix. In my case I wanted to send messages between tauri and actix so I kept looking. I found [this](https://github.com/mihai-dinculescu/msfs-2020-gps-link/commit/30fe647c6e370382665906878bdccfbc53918e37#diff-89a3f09068efd5fab83613a3a48d05162e6c098ca1975342f272a599e4a66ee9R28) approach. Sharing just in case it would be inspirational.

**Comment by joepio** [comment]
Thanks for sharing, @iilyak ! I ended up [doing this](https://github.com/joepio/atomic-data-rust/blob/master/src-tauri/src/main.rs), which means spinning up a runtime in a non-blocking Tauri async runtime.

**Comment by joepio** [comment]
I'll keep this issue on hold as Tauri is no longer functional. Blocked by: https://github.com/joepio/atomic-data-rust/issues/246

**Comment by joepio** [comment]
Tauri is working again, so this issue wont be on hold anymore.
